src/tests/TestConnection.java

```diff
@@ -0,0 +1,23 @@
+package tests;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class TestConnection {
+	
+	public static void main(String[] args) throws SQLException {
```

Comment:
Instead of this main method please create a method that returns the connection.

```java
	public Connection getConnection() {
		...
		return con;
	}
```